sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_configure.py

```diff
@@ -0,0 +1,210 @@
+    :keyword Sequence[View] views: Specifies the list of views to configure for the metric pipeline.
+    :keyword float sampling_ratio: Specifies the ratio of distributed tracing telemetry to be sampled. Accepted values are in the range [0,1]. Defaults to 1.0, meaning no telemetry is sampled out.
+    :keyword int tracing_export_interval_ms: Specifies the distributed tracing export interval in milliseconds. Defaults to 5000.
+    :keyword InstrumentationConfig instrumentation_config: Specifies a dictionary of kwargs that will be applied to instrumentation configuration. You can specify which instrumentation you want to
+        configure by name in the key field and value as a dictionary representing `kwargs` for the corresponding instrumentation.
+        Refer to the `Supported Library` section of https://github.com/microsoft/ApplicationInsights-Python/tree/main/azure-monitor-opentelemetry#officially-supported-instrumentations for the list of suppoprted library names.
+    :keyword bool disable_offline_storage: Boolean value to determine whether to disable storing failed telemetry records for retry. Defaults to `False`.
+    :keyword str storage_directory: Storage directory in which to store retry files. Defaults to `<tempfile.gettempdir()>/Microsoft/AzureMonitor/opentelemetry-python-<your-instrumentation-key>`.
+    :rtype: None
+    """
+
+    configurations = _get_configurations(**kwargs)
+
+    disable_tracing = configurations[DISABLE_TRACING_ARG]
+    disable_logging = configurations[DISABLE_LOGGING_ARG]
+    disable_metrics = configurations[DISABLE_METRICS_ARG]
+
+    resource = None
+    if not disable_logging or not disable_tracing or not disable_metrics:
+        resource = _get_resource(configurations)
+
+    # Setup tracing pipeline
+    if not disable_tracing:
+        _setup_tracing(resource, configurations)
+
+    # Setup logging pipeline
+    if not disable_logging:
+        _setup_logging(resource, configurations)
+
+    # Setup metrics pipeline
+    if not disable_metrics:
+        _setup_metrics(resource, configurations)
+
+    # Setup instrumentations
+    # Instrumentations need to be setup last so to use the global providers
+    # instanstiated in the other setup steps
+    _setup_instrumentations(configurations)
+
+
+def _get_resource(configurations: Dict[str, ConfigurationValue]) -> Resource:
+    return configurations.get(RESOURCE_ARG, Resource.create())
+
+
+def _setup_tracing(
+    resource: Resource, configurations: Dict[str, ConfigurationValue]
+):
+    sampling_ratio = configurations[SAMPLING_RATIO_ARG]
+    tracing_export_interval_ms = configurations[TRACING_EXPORT_INTERVAL_MS_ARG]
+    tracer_provider = TracerProvider(
+        sampler=ApplicationInsightsSampler(sampling_ratio=sampling_ratio),
+        resource=resource,
+    )
+    set_tracer_provider(tracer_provider)
+    trace_exporter = AzureMonitorTraceExporter(**configurations)
```

Comment:
@annatisch Is it okay to pass through any and all kwards to the exporter which is in preview? What if we only mention we support a static list of args in docs? If still no, I can remove this and instead directly hardcode the connections string and credential.